contracts/TokenRulesMock.sol

```diff
@@ -5,7 +5,25 @@ contract TokenRulesMock {
     /* Storage */
 
     mapping (address => bool) public allowedTransfers;
+    address public from;
+    address[] public transferTo;
+    uint256[] public transferAmount;
+    bool public called;
```

Comment:
If we are going to set from, transferTo and transferAmount then we don't need to have called.

Reply:
Done